scripts/styleguide-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
I would probably `chmod a+x scripts/*` and then drop the `.sh` extension. But that's just a matter of taste.